src/main/resources/network/brightspots/rcv/hints_cvr_files.txt

```diff
@@ -4,7 +4,12 @@ The tabulator needs to know where each of your CVR files is and how to interpret
 
 For each of your CVR files, provide the necessary information and then use the Add button to add it to the list.
 
-Provider (required): The vendor/machine that generated (produced) the file. After you select the field, the tabulator will fill in as many of the other fields as it can based on what it knows about that provider. You can adjust those values as necessary.
+Provider (required): The vendor/machine that generated (produced) the file. After you select the field, the tabulator will fill in as many of the other fields as it can based on what it knows about that provider. You can adjust those values as necessary. The options are:
+  * Dominion. Example: https://github.com/BrightSpots/rcv/tree/develop/src/test/resources/network/brightspots/rcv/test_data/dominion_alaska/
+  * ES&S. Example: https://github.com/BrightSpots/rcv/blob/develop/src/test/resources/network/brightspots/rcv/test_data/aliases_ess_xlsx/
+  * Hart. Example: https://github.com/BrightSpots/rcv/tree/develop/src/test/resources/network/brightspots/rcv/test_data/hart_cedar_park_school_board/
+  * CSV. Example: https://github.com/BrightSpots/rcv/tree/develop/src/test/resources/network/brightspots/rcv/test_data/generic_csv_test/
```

Comment:
See notes from other conversation related to this -- I think we should remove these long example links and either replace with a single link to the Wiki at the bottom or, maybe better yet (since we won't have to worry about keeping it up to date), not link to examples at all.

Also, it seems like you're missing Clear Ballot here, so please be comprehensive even if it's just adding the word alone with the provider name (and maybe just CSV has a bit of an explanation after it per the other thread).

Reply:
Yeah, I think that's great, thanks for setting that up! Only caveat is that the Qualtrics comment might not apply yet because you have to enable the checkbox that inverts the CSV format, right? Fine to leave as-is if you intend to add that feature as part of 1.4.0.